os/net/lwip/src/api/sockets.c

```diff
@@ -2947,9 +2980,9 @@ static int lwip_socket_register_membership(int s, const ip4_addr_t *if_addr, con
  *
  * ATTENTION: this function is called from tcpip_thread (or under CORE_LOCK).
  */
-static void lwip_socket_unregister_membership(int s, const ip4_addr_t *if_addr, const ip4_addr_t *multi_addr)
+static void lwip_socket_unregister_membership(int s, const ip4_addr_t *if_addr, const ip4_addr_t *multi_addr, pid_t pid)
```

Comment:
`pid` is not used.
I have queries for that.
1. Is it ok to treat (or handle) other's socket by pid value?
2. If 1st is wrong, how about moving `getpid` into `get_socket` inside?

Reply:
pid is used now.